Automattic/jetpack
# E2E tests: optimize scheduled runs

#### Changes proposed in this Pull Request:
Because Jetpack tests are always included, no matter the changed files, they incorrectly get executed on `scheduled` events. With this PR, the target projects and changed files are ignored for scheduled events.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run `./.github/files/create-e2e-projects-matrix.sh` script with different GITHUB_EVENT_NAME and CRON env variables values. 
    * For  GITHUB_EVENT_NAME=schedule only Jetpack on Atomic or Jetpack with Gutenberg should run, depending on CRON value
